ports/workflow/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "workflow",
-  "version-string": "0.9.3",
+  "version-string": "0.9.4",
```

```suggestion
  "version": "0.9.4",
```
This looks like a numeric+dots version, so we should use that scheme to enable versioning support